README.md

```diff
@@ -254,3 +254,22 @@ associated e-mail address.
 
 [web demo]: http://bakaoh.com/sss-wasm/
 [randombytes]: https://github.com/dsprenkels/randombytes
+
+### Endnotes
```

Comment:
At this point in the document, we are still in the F.A.Q. section. Can you put this subsection before `Other`, and change the title to a question?

Maybe something like `Can you recommend other Shamir secret sharing software?`.

Reply:
Okay, but would it be too long?

Reply:
It might be. You can abbreviate `Shamir secret sharing` or leave it out if you like?

Maybe `Is there other SSS software out there?`, or `Are there other implementations aside from this library?`

Feel free to be creative. 😄